radio/src/translations/de.h

```diff
@@ -888,29 +888,29 @@
 #define TR_AND_SWITCH                  "UND Schalt" // UND mit weiterem Schaltern
 #define TR_SF                          "SF" // Spezial Funktionen
 #define TR_GF                          "GF" // Globale Funktionen
-#define TR_ANADIAGS_CALIB              "Calibrated analogs"
-#define TR_ANADIAGS_FILTRAWDEV         "Filtered raw analogs with deviation"
-#define TR_ANADIAGS_UNFILTRAW          "Unfiltered raw analogs"
-#define TR_ANADIAGS_MINMAX             "Min., max. and range"
+#define TR_ANADIAGS_CALIB              "analoge Geber Kalibriert"
+#define TR_ANADIAGS_FILTRAWDEV         "analoge Geber gefiltert und unbearbeitet mit Abweichungen"
+#define TR_ANADIAGS_UNFILTRAW          "analoge Geber ungefiltert und unbearbeitet"
+#define TR_ANADIAGS_MINMAX             "Min., Max. und Bereich"
 #define TR_ANADIAGS_MOVE               "Move analogs to their extremes!"
 #define TR_SPEAKER                     INDENT "Lautspr"
 #define TR_BUZZER                      INDENT "Summer"
 #define TR_BYTES                       "Bytes"
-#define TR_MODULE_BIND                 BUTTON(TR("Bnd","Bind"))   //9XR-Pro
-#define TR_POWERMETER_ATTN_NEEDED      "Attenuator needed"
-#define TR_PXX2_SELECT_RX              "Select RX"
+#define TR_MODULE_BIND                 BUTTON(TR("Bnd","Binde"))   //9XR-Pro
```

Comment:
Is this the label of the "bind" button? If so, I would prefer "Binden", "Binde" sounds more of a status than an action.

Reply:
Yes it's the Bind Button.
done